chain/client/tests/chunks_management.rs

```diff
@@ -100,8 +100,7 @@ fn update_chunk_height_created(
 fn store_partial_encoded_chunk_sanity() {
     init_test_logger();
     let mut env = TestEnv::new(ChainGenesis::test(), 1, 1);
-    let signer =
-        InMemoryValidatorSigner::from_seed("test0".parse().unwrap(), KeyType::ED25519, "test0");
+    let signer = InMemoryValidatorSigner::from_seed("test0", KeyType::ED25519, "test0");
```

Comment:
Revert this change, it is expected to have `.parse().unwrap()` since we introduced strictly typed AccountId #4621

Reply:
Done.
It took github a few hours to show it as reverted due to a github incident yesterday.